web/controllers/respondent_controller.ex

```diff
@@ -85,6 +87,37 @@ defmodule Ask.RespondentController do
       |> present_fields("response")
   end
 
+  defp partial_relevant_fields(respondents) do
+    if has_any_respondent_any_partial_relevant_question?(respondents) do
```

Comment:
whether to include the partial relevant field or not should depend only on the configuration of the quex. 
* It does not matter if there are or not respondents with relevant questions answered.
* It does not matter how many relevant questions the quex has

Only if the quex toggle is selected.

Reply:
"It does not matter if there are or not respondents with relevant questions answered" -> I had this in mind.
"It does not matter how many relevant questions the quex has" -> OK, I see that if the toggle is ON, we expect that the questionnaire has at least one relevant question. I'll change that.

Reply:
If some quex of the survey has the toggle of partial relevant, then the field should appear on all respondents of the survey.

When computing the field value for the respondent, use the quex to determine if/which are the partial relevant questions.

Reply:
OK, I understand the intention. I'll review the code to see what change is needed to reach that. Thanks!!